myuw/dao/myuw_notice.py

```diff
@@ -1,60 +1,65 @@
 from myuw.dao.affiliation import get_all_affiliations
 from myuw.models.myuw_notice import MyuwNotice
-from myuw.dao.term import get_comparison_datetime
+from myuw.dao.term import get_comparison_datetime_with_tz
 
 
 def get_myuw_notices_for_user(request):
-    date = get_comparison_datetime(request)
+    date = get_comparison_datetime_with_tz(request)
```

Comment:
For RuntimeWarning: DateTimeField MyuwNotice.end received a naive datetime (2018-09-07 06:13:00) while time zone support is active